src/js/components/CheckBox/stories/WithStickyDiv.js

```diff
@@ -1,61 +1,58 @@
-import React, { Component } from 'react';
-import { storiesOf } from '@storybook/react';
+import { Box, CheckBox, Grommet } from 'grommet';
+import React, { useState } from 'react';
 
-import { Box, Grommet, CheckBox } from 'grommet';
 import { grommet } from 'grommet/themes';
+import { storiesOf } from '@storybook/react';
+
+const boxBackground = { color: 'neutral-1' };
+const boxStyle = { position: 'sticky', top: 0 };
+
+const checkboxes = Array(8)
+  .fill()
+  .map((_, i) => `item ${i + 1}`);
+
+const removeItemFromArray = (array, value) =>
+  array.filter(item => item !== value);
 
-class CheckBoxWithStickyDiv extends Component {
-  state = {
-    checked: [],
-    checkboxes: Array(8)
-      .fill()
-      .map((_, i) => `item ${i + 1}`),
-  };
-
-  onCheck = (event, value) => {
-    const { checked } = this.state;
-    if (event.target.checked) {
-      checked.push(value);
-      this.setState({ checked });
-    } else {
-      this.setState({ checked: checked.filter(item => item !== value) });
-    }
-  };
-
-  render() {
-    const { checked, checkboxes } = this.state;
-    return (
-      <Grommet theme={grommet}>
-        <Box pad="large" align="center">
-          <Box
-            height="120px"
-            width="120px"
-            overflow="auto"
-            style={{
-              position: 'relative',
-              display: 'block',
-            }}
-          >
-            <Box
-              background={{ color: 'neutral-1' }}
-              style={{ position: 'sticky', top: 0 }}
-            >
-              Click &amp; Scroll
-            </Box>
-            {checkboxes.map(item => (
-              <CheckBox
-                key={item}
-                checked={checked.indexOf(item) !== -1}
-                label={item}
-                onChange={e => this.onCheck(e, item)}
-              />
-            ))}
+const getChecks = ({ target }, checked, value) => {
```

Comment:
can we add those functions inside the story. maybe in a normal code you want this separated but lets help the users see and get the code they need easier in storybook. plus i don't see the reason of creating a function to get checks. if you leave it inside the component then you can do that directly.